creator-node/src/index.ts

```diff
@@ -89,15 +80,61 @@ const startApp = async () => {
     throw new Error('Invalid delegatePrivateKey/delegateOwnerWallet pair')
   }
 
+  // Fail if Trusted Notifier isn't configured properly
   const trustedNotifierEnabled = !!config.get('trustedNotifierID')
   const nodeOperatorEmailAddress = config.get('nodeOperatorEmailAddress')
-
   if (!trustedNotifierEnabled && !nodeOperatorEmailAddress) {
     exitWithError(
       'Cannot startup without a trustedNotifierID or nodeOperatorEmailAddress'
     )
   }
 
+  await connectToDBAndRunMigrations()
+
+  // Clear all redis locks
+  try {
+    await redisClient.WalletWriteLock.clearWriteLocks()
+  } catch (e: any) {
+    logger.warn(`Could not clear write locks. Skipping..: ${e.message}`)
+  }
+
+  // This is called `cpus()` but it actually returns the # of logical cores, which is possibly higher than # of physical cores if there's hyperthreading
+  const logicalCores = cpus().length
+  const numWorkers = config.get('expressAppConcurrency') || logicalCores
+  logger.info(`Spawning ${numWorkers} processes to run the Express app...`)
+  _.times(numWorkers, () => cluster.fork())
+
+  cluster.on('exit', (worker, code, signal) => {
+    logger.info(`Worker process with pid=${worker.process.pid} died`)
+  })
+}
+
+// Workers don't share memory, so each one is its own Express instance with its own version of objects like serviceRegistry
+const startAppForWorker = async () => {
```

Comment:
worker might need to do some of the stuff from primary e.g.
```
if (!delegateOwnerWallet || !delegatePrivateKey || !creatorNodeEndpoint) {
    exitWithError(
      'Cannot startup without delegateOwnerWallet, delegatePrivateKey, and creatorNodeEndpoint'
    )
  }
```
it'll need those in memory right

Reply:
that's just checking the config. it runs the code with the same config as the primary/master, so as long as the primary/master checked that the config values are good then the workers will be ok

Reply:
but every worker has its own config.js with separate memory, so should check in each?

Reply:
it's basically a clone so yeah the memory is different but the config.js can't possibly change between the primary and workers spawning. as long as one checked config.js then it will be safe for all of them

Reply:
probably - but its a free check and would eliminate hard-to-test issues down the line if we do this redundant work to ensure each worker is fully initialized

Reply:
alright I extracted out some checks and made the workers each run them

Reply:
sick